### ISSUE TITLE
Does pa11y-dashboard provide monitoring?

### ISSUE THREAD

**Issue by anevins12** [pa11y/pa11y-dashboard#259]
I understand that a test can be run manually, but I was wondering whether a test could be ran daily or weekly for example.

**Comment by josebolos** [comment]
Hi @anevins12,

Yes, the goal of pa11y-dashboard is to track the a11y violations over time and by default tests will run once a day, although this can be configured when deploying the app, e.g.:
https://github.com/pa11y/pa11y-dashboard/blob/master/config/production.sample.json#L10

The last 30 days of data will show in the graphs, unfortunately this is not configurable.

I realise that the documentation doesn't explain this very well, so we definitely should improve this.

**Comment by anevins12** [comment]
Thank you for coming back to me Jose, I had read it somewhere but on looking at the readme I thought I may have made a mistake. Thanks for clarifying that it does carry out testing over time, I appreciate all the work into this!

**Comment by josebolos** [comment]
Thanks for creating this issue. It looks like this may be fixed now, feel free to reopen the issue if not.

**Comment by andreabisello** [comment]
@anevins12 in order to use cron properties we must to be in a linux machine with cron?

**Comment by anevins12** [comment]
CC @josebolos

**Comment by josebolos** [comment]
@andreabisello No, Pa11y Dashboard doesn't make use of the cron tool.
